### DISCUSSION TITLE
Retrieve from multiple collections (blade)

### DISCUSSION THREAD

**Discussion by spaceageliving** [statamic/cms#6609] [Troubleshooting] [answered]
I'm using a collection tag like this in my blade view:

```
@foreach(Statamic::tag('collection:features') as $entry)
...
@endforeach
```

Is it possible to use this approach to pull from multiple collections? I've tried passing an array:

```
@foreach(Statamic::tag(['collection:collection1', 'collection:collection2']) as $entry)
```

and also a list:

```
@foreach(Statamic::tag('collection:collection1', 'collection:collection2') as $entry)
```

but neither are working. Am I missing something, or is this just not possible?

**Comment by jasonvarga** [comment, marked as answer]
The `Statamic::tag()` mimics the Antlers tags.

To do multiple collections in Antlers, you have to do `{{ collection from="one|two" }}`.

So, in Blade you should do `Statamic::tag('collection')->from('one|two')`

**Reply by spaceageliving** [reply]
@jasonvarga You are awesome thanks for the quick response.

Is it also possible to paginate and sort with multiple collections?

**Reply by jasonvarga** [reply]
Yep!

**Comment by spaceageliving** [comment]
If you guys keep this up, people are gonna start talking ;)
